lib/neovim-plugin.nix

```
@@ -157,7 +166,55 @@
                 ])
                 ++ (lib.optionals hasConfigAttrs [
                   (lib.optionalAttrs callSetup { ${namespace}.${name}.luaConfig.content = setupCode; })
-                  (lib.optionalAttrs (configLocation != null) (setLuaConfig cfg.luaConfig.content))
+                  (lib.mkIf (!cfg.lazyLoad.enable) (
+                    lib.optionalAttrs (configLocation != null) (setLuaConfig cfg.luaConfig.content)
+                  ))
+                ])
+                ++ (lib.optionals hasConfigAttrs [
```

```suggestion
                ++ lib.optionals hasConfigAttrs [
```
:sob: Maybe it's just me, but I actually find it easier to read when there's less parens; it makes the remaining parens more meaningful